torch/distributed/_tensor/debug/comm_mode.py

```diff
@@ -29,6 +29,7 @@
     c10d_ops._allgather_base_,
     c10d_ops._reduce_scatter_base_,
     c10d_ops.broadcast_,
+    c10d_ops.allreduce_coalesced_,
```

Comment:
the list is not kept in alphabetic order. This can be a TODO in next PR to make it in order.